lib/src/blocs/account_page_bloc.dart

```diff
@@ -0,0 +1,15 @@
+import 'dart:async';
+
+import 'package:reazzon/src/chat/base_bloc/base_bloc.dart';
+
+class AccountPageBloc extends BLOCBase {
```

Comment:
BLOCBase it's confusing naming. I think it should be named BlocEventStateBase, see my other comment where I linked the gist file

Reply:
Or you might want to just use the actual BlocBase, like you used in the FilterBloc

Reply:
The implementation will be more confusing.
The BLOCBase I created is only abstraction of the same base, that creates the stream itself and dispose them.